js/app/src/api_docs/CodeSnippets.svelte

```diff
@@ -137,10 +133,7 @@ async function run() &lbrace;
 						{/each}
 	]);
 
-	console.log(result?.data);
-&rbrace;
-
-run();
+console.log(result.data);
```

Comment:
Don't we need to keep `result?.data`

Reply:
I don't think it's necessary but @pngwn could you confirm?

Reply:
Will revert unless I hear otherwise from @pngwn